config.json

```diff
@@ -528,9 +528,10 @@
       "slug": "diamond",
       "core": false,
       "unlocked_by": null,
-      "difficulty": 1,
+      "difficulty": 7,
```

Comment:
I agree. Since tests seem involving due to how they are written (property-based) but solution to exercise isn't that difficult.